ui/src/app/utils/access-utils.tsx

```diff
@@ -545,34 +546,65 @@ export const isExpiringOrExpired = (
   return !!expiration && getWholeDaysFromNow(expiration) <= lookback;
 };
 
+export const isCompleted = (
```

Comment:
Some of these functions in this file have been reordered, to comply with "don't call before defining"